API.md

```diff
@@ -38,6 +38,8 @@ some relevant data and/or a `message` field.
   * [`GET /api/hugo/admin/ballots/:category`](#get-apihugoadminballotscategory)
   * [`GET /api/hugo/admin/canon`](#get-apihugoadmincanon)
   * [`POST /api/hugo/admin/canon/:id`](#post-apihugoadmincanonid)
+  * [WebSocket: `wss://server/api/hugo/admin/canon-updates`](#websocket-wssserverapihugoadmincanonupdates)
+* [Raami exhibitions](#raami)
```

Comment:
Should be at the end, not in the middle of the hugo-admin links. Also, the title link should be to `#raami-exhibition` and the individual endpoints included here.